packages/xod-client/src/core/styles/components/Debugger.scss

```diff
@@ -1,7 +1,7 @@
 .debug-session-stop-button {
   z-index: 10;
   position: absolute;
-  top: 3px;
+  top: 33px;
```

Comment:
It looks quite fragile. I think the better solution is allowing `BreadcrumbsContainer` to have children which will be flex-aligned to the right. The same will help when we’d add buttons for the embedded code editor.

What do you think?